frontend/src/App.js

```diff
@@ -2,13 +2,13 @@ import React, { useState, useEffect } from "react";
 import { getBlockchain } from "./ethereum.js";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import { makeStyles } from '@material-ui/core/styles';
-import Layout from "./components/layout/Layout.jsx";
```

Comment:
we should avoid using file extensions on import statements. It's unnecessary and webpack won't be able to find the file if we migrate it to typescript.